### ISSUE TITLE
Cannot start wallet

### ISSUE THREAD

**Issue by mastercyb** [cloudhead/nakamoto#82]
in terminal
```
./nakamoto-wallet --addresses bc1q0qjylfgu4t8yky6fxr599yw23ruqun0030t9af --genesis 000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f
```
gives this
```
Error parsing option '--genesis' with value '000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f': invalid digit found in string
```

What could I do wrong?

**Comment by metaclips** [comment]
genesis is the blockchain “height” you want to start scanning from

**Comment by vincenzopalazzo** [comment]
or maybe `--start-from-height `or only ` --start-height`?

**Comment by mastercyb** [comment]
`--birth-height` seems really cool

**Comment by cloudhead** [comment]
Closed in 5862aab
